coin-flip/script.js

```diff
@@ -2,12 +2,11 @@ function getRandNumber(max) {
 	return Math.floor(Math.random() * max);
 }
 
-var number = getRandNumber(2);
 
-var headsOrTails = "None";
+const flipCoin = () => {
```

Comment:
Current JS standards are to use `let` (mutable) and `const` (immutable). "Mutable" means you can change the value of the variable later on.

Now, if you're dealing with an array (for example), you can always change the values _inside_ the array, you just can't reassign the variable itself to something else.